rgsync/Connectors/__init__.py

```diff
@@ -1,11 +1,15 @@
 from .simple_hash_connector import SimpleHashConnector
-from .sql_connectors import MsSqlConnector, MySqlConnector, SQLiteConnection, OracleSqlConnector, SnowflakeSqlConnector, MsSqlConnection, MySqlConnection, OracleSqlConnection, SnowflakeSqlConnection, SQLiteConnector
+from .sql_connectors import MsSqlConnector, MySqlConnector, SQLiteConnection, OracleSqlConnector, SnowflakeSqlConnector, MsSqlConnection, MySqlConnection, OracleSqlConnection, SnowflakeSqlConnection, SQLiteConnector, BaseSqlConnection, BaseSqlConnector, DB2Connection, DB2Connector
```

Comment:
At this point do we have any reason to not just import *?

Reply:
Nope, you are right, will fix it.